docker/Dockerfile

```diff
@@ -1,13 +1,15 @@
 FROM --platform=$BUILDPLATFORM golang:1.19 as builder
-RUN apt-get install make
 WORKDIR /src
 COPY go.mod go.sum ./
 RUN go mod download
 COPY . .
+
 ARG TARGETOS TARGETARCH
-RUN env GOOS=$TARGETOS GOARCH=$TARGETARCH make build
+ENV GOOS=$TARGETOS
+ENV GOARCH=$TARGETARCH
+RUN make build
 
-FROM ubuntu
+FROM ubuntu:20.04
```

Comment:
I assume no version means the latest. Why not the latest as everywhere else?

Reply:
From a source control standpoint, it is better to be explicit.  Despite the best intentions, bugs get released, and we don't want to waste time debugging some bug in a dependency.  Better to explicitly upgrade so we can more easily determine when bugs are introduced.

Reply:
My question mostly comes from the usage of latest for the CI jobs, but ok